ouroboros-consensus/src/mempool-test-utils/Test/Consensus/Mempool/Mocked.hs

```diff
@@ -27,79 +26,76 @@ import qualified Ouroboros.Consensus.Mempool as Mempool
 import           Ouroboros.Consensus.Mempool.API (AddTxOnBehalfOf,
                      MempoolAddTxResult)
 
-data MempoolWithMockedLedgerItf m blk = MempoolWithMockedLedgerItf {
+data MockedMempool m blk = MockedMempool {
       getLedgerInterface :: !(Mempool.LedgerInterface m blk)
     , getLedgerStateTVar :: !(StrictTVar m (LedgerState blk))
     , getMempool         :: !(Mempool m blk)
     }
 
-instance NFData (MempoolWithMockedLedgerItf m blk) where
+instance NFData (MockedMempool m blk) where
   -- TODO: check we're OK with skipping the evaluation of the
-  -- MempoolWithMockedLedgerItf. The only data we could force here is the
+  -- MockedMempool. The only data we could force here is the
   -- 'LedgerState' inside 'getLedgerStateTVar', but that would require adding a
   -- 'NFData' constraint and perform unsafe IO. Since we only require this
   -- instance to be able to use
   -- [env](<https://hackage.haskell.org/package/tasty-bench-0.3.3/docs/Test-Tasty-Bench.html#v:env),
   -- and we only care about initializing the mempool before running the
   -- benchmarks, maybe this definition is enough.
-  rnf MempoolWithMockedLedgerItf {} = ()
+  rnf MockedMempool {} = ()
 
-data InitialMempoolAndModelParams blk = MempoolAndModelParams {
-      immpInitialState :: !(Ledger.LedgerState blk)
-    , immpLedgerConfig :: !(Ledger.LedgerConfig blk)
-    }
```

Comment:
In `feature/utxo-hd`, [these params are extended with 2 additional fields](https://github.com/input-output-hk/ouroboros-consensus/blob/feature/utxo-hd/ouroboros-consensus/bench/mempool-bench/Bench/Consensus/Mempool/Params.hs#L87-L94). Maybe we can keep `InitialMempoolAndModelParams`  and/or rename it?

Reply:
Good point. I'll revert this.